src/mixer/plugins/volume_mapping.cxx

```diff
@@ -43,6 +41,7 @@
 #endif /* __UCLIBC__ */
 
 #define MAX_LINEAR_DB_SCALE	24
+#define SND_CTL_TLV_DB_GAIN_MUTE   -9999999
```

Comment:
what do you mean again? Where is it defined? Doing this fixed my issue, as min on my card actually is -9999999